pkg/lib/notifications.ts

```diff
@@ -115,28 +115,34 @@ Usage:
 
 */
 
-import cockpit from "cockpit";
+import cockpit, { JsonValue, JsonObject } from "cockpit";
 import { dequal } from 'dequal/lite';
 
-class PageStatus {
+export interface Status {
```

Comment:
The parallel shell PR landed, so this supersedes the declaration in pkg/shell/state.tsx . I'm fine if you want to do that as a follow-up of course.

Reply:
Let's do it here.

Reply:
Done